src/DssSpell.t.sol

```diff
@@ -490,12 +490,12 @@ contract DssSpellTest is DSTest, DSMath {
     function checkCollateralValues(bytes32 ilk, SystemValues storage values) internal {
         (uint duty,)  = jug.ilks(ilk);
 
-        assertEq(duty, rates.rates(values.collaterals[ilk].pct / 10));
+        assertEq(duty, rates.rates(values.collaterals[ilk].pct));
         // make sure duty is less than 1000% APR
         // bc -l <<< 'scale=27; e( l(10.00)/(60 * 60 * 24 * 365) )'
         // 1000000073014496989316680335
         assertTrue(duty >= RAY && duty < 1000000073014496989316680335);  // gt 0 and lt 1000%
-        assertTrue(diffCalc(expectedRate(values.collaterals[ilk].pct), yearlyYield(rates.rates(values.collaterals[ilk].pct / 10))) <= TOLERANCE);
+        assertTrue(diffCalc(expectedRate(values.collaterals[ilk].pct * 10), yearlyYield(rates.rates(values.collaterals[ilk].pct))) <= TOLERANCE);
```

Comment:
Would be great if we could use basis points all the way down, but not a big deal if it's inconvenient for some reason.

Reply:
Same comment as on the other PR:

Sorry, looks like I didn't communicate clearly enough--I just meant that it was very inelegent to be multiplying pct by 10 here, so it would be nice if expectedRate could just expect a basis point argument, I didn't mean to use basis points to express all values.

Reply:
Pushed a change to refactor this.